src/commonControls/multiValueListControl/MultiValueListControl.ts

```diff
@@ -714,6 +715,11 @@ export class MultiValueListControl extends Control implements InteractionModelCo
             canHandle: this.isRemoveWithValue,
             handle: this.handleRemoveWithValue,
         },
+        {
+            name: 'RemoveWithOrdinal (builtin)',
```

Comment:
Also would be great if you can add a test case for the removal of ordinal with order :)